src/iqm/cortex_cli/auth.py

```diff
@@ -78,8 +79,8 @@ def login_request(url: str, realm: str, client_id: str, username: str, password:
 
     data = AuthRequest(client_id=client_id, grant_type=GrantType.PASSWORD, username=username, password=password)
 
-    request_url = f'{url}/realms/{realm}/protocol/openid-connect/token'
-    result = requests.post(request_url, data=data.dict(exclude_none=True), timeout=AUTH_REQUESTS_TIMEOUT)
+    request_url = urljoin(url, f'/realms/{realm}/protocol/openid-connect/token')
```

Comment:
This is because tests failed due to a double slash that appeared I guess due to Pydantic parsing of URLs.